plugins/scaffolder-backend/api-report.md

```diff
@@ -712,6 +725,13 @@ export type TaskEventType = 'completion' | 'log';
 
 // @public
 export class TaskManager implements TaskContext {
+  // (undocumented)
+  get abortContext(): {
+    abort: () => Promise<void>;
+    abortListener: () => void;
+    setAbortListener: (listener: () => void) => void;
+    signal: AbortSignal;
+  };
```

Comment:
An `AbortSignal` already is an `EventTarget`, which means it can have event listeners. Seems to me that the `abortListener` and `setAbortListener` fields might be redundant, since you would instead just do `.signal.on('abort', () => ...)`